app/workers/functions.php

```diff
@@ -74,40 +63,17 @@
 
 $executionStart = \microtime(true);
 
-$exitCode = Console::execute('docker ps --all --format "name={{.Names}}&status={{.Status}}&labels={{.Labels}}" --filter label=appwrite-type=function'
-    , '', $stdout, $stderr, 30);
-
-$executionEnd = \microtime(true);
+$response = $orchestration->list(['label' => 'appwrite-type=function']);
 
 $list = [];
-$stdout = \explode("\n", $stdout);
-
-\array_map(function($value) use (&$list) {
-    $container = [];
-
-    \parse_str($value, $container);
 
-    if(isset($container['name'])) {
-        $container = [
-            'name' => $container['name'],
-            'online' => (\substr($container['status'], 0, 2) === 'Up'),
-            'status' => $container['status'],
-            'labels' => $container['labels'],
-        ];
+array_map(function($value) use (&$list) {
```

Comment:
I wouldn't use array map if you are not returning any value in the callback. Either switch to `foreach` or return in the callback and assign the new value.